docs/vector.rst

```diff
@@ -13,25 +13,25 @@ In fact, ``Vector`` is simply an alias of ``Matrix``, just with ``m = 1``. See :
 Declaration
 -----------
 
-As global fields of vectors
-+++++++++++++++++++++++++++
+As global vector fields
++++++++++++++++++++++++
 
 .. function:: ti.Vector.field(n, dtype, shape = None, offset = None)
 
     :parameter n: (scalar) the number of components in the vector
     :parameter dtype: (DataType) data type of the components
-    :parameter shape: (optional, scalar or tuple) shape the field of vectors, see :ref:`tensor`
+    :parameter shape: (optional, scalar or tuple) shape of the vector field, see :ref:`tensor`
     :parameter offset: (optional, scalar or tuple) see :ref:`offset`
 
-    For example, this creates a 5x4 field of 3 component vectors:
+    For example, this creates a ``5x4`` vector field which has a 3 component vector on each entry:
     ::
 
         # Python-scope
         a = ti.Vector.field(3, dtype=ti.f32, shape=(5, 4))
 
 .. note::
 
-    In Python-scope, ``ti.field`` declares :ref:`scalar_tensor`, while ``ti.Vector.field`` declares fields of vectors.
+    In Python-scope, ``ti.field`` declares :ref:`scalar_tensor`, while ``ti.Vector.field`` declares vector fields.
```

Comment:
```suggestion
    In Python-scope, ``ti.field`` declares a :ref:`scalar_tensor`, while ``ti.Vector.field`` declares a vector field.
```

Also, are we modifying the term 'scalar_tensor' into 'scalar_field' in another repo? :> so we don't have to correct here?

Reply:
Thanks for this suggestion! We have modified all tensor -> field in previous PR. Honesely, here the `scalar_tensor` is a filename, If I rename it to `scalar_field` it would cause a quite big changeset which maybe we dislike.

Hopefully this rename work can be done later, HDYT?

Reply:
No problem, just want to know if someone is on this. :> great job!

Reply:
How about rename `scalar_tensor` to just `field`? IIRC `ti.Vector.field` is also based on `ti.field`'s so the name `field.rst` is OK for simplicity.

Reply:
Thanks for your suggestion! Just now I take a brief look at `scalar_tensor.rst`, it seems that this document try to illustrate the usage and property of `field`, more specified, `scalar field`. 

When we rename `scalar_tensor` to `field`, it would be kinds of confusing, because it can not tell us whether it is `scalar field`, `vector field` or `matrix`. How about renaming it from `scalar_tensor` to `scalar_field` IMO? It contains `field`, and are more focused.

Maybe I can do this renaming when this PR are **nearly finished**. otherwise it will cause a too big changeset to be reviewed.

Reply:
I am more with `scalar_field` as well. :>

Reply:
> I am more with `scalar_field` as well. :>

Yes, let's discuss it together further more. ;-)